scripts/test_garage_deploy_deb.sh

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
-set -exuo pipefail
+set -exu
```

Comment:
I switched from bash to sh because I was trying to run them under a docker without bash. And this sh didn't have the option. Doesn't matter as there is no pipe in this script :).

But in the end I've changed my strategy and moved most of it inside the gitlab yaml as it took as much boilerplate to invoke this script properly rather than just hard-code it.